core/app/[locale]/(default)/product/[slug]/page.tsx

```diff
@@ -318,6 +330,13 @@ export default async function Product(props: Props) {
           </>
         )}
       </Stream>
+
+      <WishlistButtonForm
+        formId={detachedWishlistFormId}
+        productId={productId}
+        productSku={streamableProductSku}
+        searchParams={props.searchParams}
+      />
```

Comment:
What is the reason for this form to not live inside `product-details-form`?

Reply:
@jorgemoya Forms can't be nested, so having it all live inside of the form and doing everything from the `addToCart` action seems unnecessarily complex since we can just have a "decoupled" form where the inputs are attached to the form ID. Seems like it will be much easier to maintain this way 👍

Reply:
I still think this should live under `ProductDetail` (and ProductDetail should have both forms). 🤔

Reply:
@jorgemoya I disagree, I still feel that we should keep it all decoupled from the VIBES component since Wishlists on the PDP is specific to BC functionality 💭  I discussed it with @migueloller already and we landed on taking the approach of having the `additionalActions` prop on `ProductDetails` so that any consumer of VIBES can pass a `ReactNode` to add stuff after the "add to cart" button. With this approach, we're able to keep everything at the top-level and decoupled from VIBES. 🤔

Reply:
Just wanted to chime in and say that we still think as VIBES being decoupled when being brought in to Catalyst so it should be perfectly fine to change the VIBES components. This is why we renamed the directory from `vibes/soul` to `ui`. That said, I do think that for VIBES it would be a good idea to have a prop that allows extending the product detail with additional actions by taking a `ReactNode`. A good use case for this, for example, is adding BNPL options without worrying about having to for the UI too much. I will admit, though, that these ideas mostly came up when we were thinking about theming from a VIBES perspective

Reply:
Another quick thought - the current approach should also make it easier for developers to migrate and update to this wishlist PDP functionality. If we move it into the ProductDetail component, it would be a bit more complicated of a migration since I think developers are more likely to have modified that component rather than the root page 💭